app/controllers/stripe-setup/responsible-person/post.controller.js

```diff
@@ -86,14 +91,21 @@ module.exports = (req, res) => {
 }
 
 const validate = (req, fieldName, fieldValidator, maxLength) => {
-  const field = getFieldValue(req, fieldName)
+  const field = lodash.get(req.body, fieldName)
   const isFieldValid = fieldValidator(field, maxLength)
   if (!isFieldValid.valid) {
     return isFieldValid.message
   }
   return null
 }
 
-const getFieldValue = (req, fieldNames) => {
-  return lodash.get(req.body, fieldNames)
+const validateDoB = (req) => {
```

Comment:
Small thing but do you need the whole `req` object? it would be best to not pass this around in general as it has a lot of information on it and can impact the Express stack.

Reply:
To be clear - recommend just passing exactly what you need, this will reduce the risk of developers not being sure how to use this object further down the line. It also means it can have a nicer name.

+ also naming methods

```suggestion
const validateDoB = function validDoB (formValues) => {
```